Dockerfile

```diff
@@ -1,49 +1,9 @@
-FROM postgres:14.2-alpine AS base
+FROM quay.io/ukhomeofficedigital/hocs-base-image
 
-ENV USER hocs
-ENV USER_ID 1000
-ENV GROUP hocs
-ENV NAME hocs-toolbox
-ENV PGDATA /app/data
-ENV AWS_CLI_VERSION 1.16.207
+USER root
 
-WORKDIR /app
+RUN apk add --no-cache postgresql-client kubectl aws-cli ca-certificates curl
 
-RUN addgroup ${GROUP} && \
-    adduser -u ${USER_ID} -G ${GROUP} -h /app -D ${USER}
-RUN mkdir -p /app/scripts && \
-    chown -R ${USER}:${GROUP} /app
-RUN mkdir -p /app/scripts
+USER 10000
```

Comment:
Consider making this one `hocs_user` or the root one further up `0` so they match.